### PR TITLE
Add italian photos

### PR DESCRIPTION
**Pull request by GresosJ**

### CODE REVIEW

File: data/places.json [hunk 1]
```diff
@@ -125,5 +125,32 @@
...
     "photo": "/photos/danielpereira/california.jpeg"
+  },
+  {
+    "author": "Gerson Júnior",
+    "type": "sticker",
+    "city": "Roma",
+    "country": "Italy",
+    "coordinates": [41.889048, 12.492502],
+    "date": "2022-07-11",
+    "photo": "/photos/gersonjunior/Coliseu.jpg"
```
Comment (nelsonmestevao) [inline data/places.json:136]: It should be your username the folder 💪

File: data/places.json [hunk 2]
```diff
@@ -125,5 +125,32 @@
...
+    "photo": "/photos/gersonjunior/Napoles.jpg"
+  },
+  {
+    "author": "Gerson Júnior",
+    "type": "sticker",
+    "city": "San Giorgio, Albanese",
+    "country": "Italy",
+    "coordinates": [39.583670, 16.452920],
+    "date": "2022-07-04",
+    "photo": "/photos/gersonjunior/SanGiorgio.jpg"
```
Comment (nelsonmestevao) [inline data/places.json:154]:
```suggestion
    "photo": "/photos/GersosJ/SanGiorgio.jpg"
```

File: data/places.json (hunk 1 above)
Comment (nelsonmestevao) [inline data/places.json:136]:
```suggestion
    "photo": "/photos/GresosJ/Coliseu.jpg"
```

File: data/places.json [hunk 3]
```diff
@@ -125,5 +125,32 @@
...
+    "photo": "/photos/gersonjunior/Coliseu.jpg"
+  },
+  {
+    "author": "Gerson Júnior",
+    "type": "sticker",
+    "city": "Napoli",
+    "country": "Italy",
+    "coordinates": [40.840007, 14.251875],
+    "date": "2022-07-08",
+    "photo": "/photos/gersonjunior/Napoles.jpg"
```
Comment (nelsonmestevao) [inline data/places.json:145]:
```suggestion
    "photo": "/photos/GresosJ/Napoles.jpg"
```

File: data/places.json (hunk 2 above)
Comment (sassypocoyo) [reply data/places.json:154]: > ```suggestion
>     "photo": "/photos/GersosJ/SanGiorgio.jpg"
> ```
> 

Shouldn't it be "GresosJ"?

File: data/places.json (hunk 1 above)
Comment (nelsonmestevao) [inline data/places.json:136]:
```suggestion
    "photo": "/photos/GresosJ/Coliseu.jpg"
```

**Review by nelsonmestevao — APPROVED** [review]

**Pull request merged**